packages/grafana-data/src/utils/location.test.ts

```diff
@@ -41,6 +41,14 @@ describe('locationUtil', () => {
       const urlWithoutMaster = locationUtil.stripBaseFromUrl('http://www.domain.com:9877/subUrl/grafana/');
       expect(urlWithoutMaster).toBe('/grafana/');
     });
+    test('relative url with subdirectory  subUrl', () => {
+      const urlWithoutMaster = locationUtil.stripBaseFromUrl('/grafana/subUrl');
+      expect(urlWithoutMaster).toBe('/grafana/subUrl');
+    });
```

Comment:
this does not look correct, should not /grafana be stripped here ?

Reply:
ah, subUrl is the real sub url here

Reply:
Yeah this has been rather confusing for me. 😄 
I've tried to change these strings in the hope it makes the tests a little clearer (possibly not).